clusterloader2/testing/experimental/storage/pod-startup/config.yaml

```diff
@@ -1,100 +1,143 @@
 # ASSUMPTIONS:
 # - Number of nodes should be divisible by NODES_PER_NAMESPACE (default 100).
 
-#Constants
-# TODO(mucahitkurt) copied from load tests, open for discussion.
-{{$NODE_MODE := DefaultParam .NODE_MODE "allnodes"}}
-{{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
-{{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 30}}
+# Cluster Variables
+  {{$NODE_MODE := DefaultParam .NODE_MODE "allnodes"}}
+  {{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
 
-{{$POD_TEMPLATE_PATH := DefaultParam .POD_TEMPLATE_PATH "../volume-types/emptydir/pod_with_emptydir.yaml"}}
-{{$LOAD_TEST_THROUGHPUT := DefaultParam .LOAD_TEST_THROUGHPUT 10}}
+# Test Variales
+  {{$TEST_NAME := DefaultParam .TEST_NAME "generic-test"}}
+  {{$PODS_PER_NODE := .PODS_PER_NODE}}
+  {{$POD_TEMPLATE_PATH := .POD_TEMPLATE_PATH }}
+  {{$VOLUMES_PER_POD := .VOLUMES_PER_POD}}
+  {{$VOLUME_TEMPLATE_PATH := .VOLUME_TEMPLATE_PATH}}
+  {{$PROVISION_VOLUME := DefaultParam .PROVISION_VOLUME false}}
+  {{$VOL_SIZE := DefaultParam .VOL_SIZE "8Gi"}}
+  {{$WAIT_FOR_PVC := DefaultParam .WAIT_FOR_PVC false}}
+  {{$POD_STARTUP_TIMEOUT := DefaultParam .POD_STARTUP_TIMEOUT "15m"}}
+  {{$POD_STARTUP_SLO := DefaultParam .POD_STARTUP_SLO 300}}
+  {{$PVC_BINDING_TIME := DefaultParam .PVC_BOUND_SLO 2}}
```

Comment:
Do we really need to expose this?

Reply:
I can remove this know for now, but we may need it later if we do other types of PD.